# Do not let povray do anything in the user home directory
# wpoely86@gmail.com
diff -ur povray-3.7.0.7.orig/unix/prebuild.sh povray-3.7.0.7/unix/prebuild.sh
--- povray-3.7.0.7.orig/unix/prebuild.sh	2018-01-05 10:06:48.000000000 +0100
+++ povray-3.7.0.7/unix/prebuild.sh	2018-04-10 10:14:08.621514093 +0200
@@ -641,19 +641,6 @@
 	for f in \$\$filelist ; do \\
 	  \$(INSTALL_DATA) \$(top_srcdir)/doc/\$\$f \$(DESTDIR)\$(povdocdir)/\$\$f && echo "\$(DESTDIR)\$(povdocdir)/\$\$f" >> \$(povinstall); \\
 	done
-	@echo "Creating user directories..."; \\
-	for p in \$(povuser) \$(povconfuser) ; do \\
-	  \$(mkdir_p) \$\$p && chown \$(povowner) \$\$p && chgrp \$(povgroup) \$\$p && printf "%s\\n" "\$\$p" "\`cat \$(povinstall)\`" > \$(povinstall); \\
-	done
-	@echo "Copying user configuration and INI files..."; \\
-	for f in povray.conf povray.ini ; do \\
-	  if test -f \$(povconfuser)/\$\$f; then \\
-	    echo "Creating backup of \$(povconfuser)/\$\$f"; \\
-	    mv -f \$(povconfuser)/\$\$f \$(povconfuser)/\$\$f.bak; \\
-	  fi; \\
-	done; \\
-	\$(INSTALL_DATA) \$(top_srcdir)/povray.conf \$(povconfuser)/povray.conf && chown \$(povowner) \$(povconfuser)/povray.conf && chgrp \$(povgroup) \$(povconfuser)/povray.conf  && echo "\$(povconfuser)/povray.conf" >> \$(povinstall); \\
-	\$(INSTALL_DATA) \$(top_builddir)/povray.ini \$(povconfuser)/povray.ini && chown \$(povowner) \$(povconfuser)/povray.ini && chgrp \$(povgroup) \$(povconfuser)/povray.ini  && echo "\$(povconfuser)/povray.ini" >> \$(povinstall)
 
 # Remove data, config, and empty folders for 'make uninstall'.
 # Use 'hook' instead of 'local' so as to properly remove *empty* folders (e.g. scripts).
